global_planner/src/nodes/global_planner_node.cpp

```diff
@@ -436,6 +438,18 @@ void GlobalPlannerNode::printPointInfo(double x, double y, double z) {
 void GlobalPlannerNode::publishSetpoint() {
   // Vector pointing from current position to the current goal
   tf::Vector3 vec = toTfVector3(subtractPoints(current_goal_.pose.position, last_pos_.pose.position));
+  if (global_planner_.use_speedup_heuristics_){
+    Cell cur_cell = global_planner::Cell(last_pos_.pose.position.x, last_pos_.pose.position.y, last_pos_.pose.position.z);
+    double cur_risk = std::sqrt(global_planner_.getRisk(cur_cell));
+    if(cur_risk >= 0.5) {   // If current risk is too high, set speed as low to stable flight.
+      speed_ = 1.0;
```

Comment:
These numbers look like magic numbers, could you parameterize these numbers somehow so it doesn't get lost in the code?

Reply:
Agreed, in different situations you might have the `global_planner_.max_speed_` to be less than 1. Rather make this a fraction of the max_speed_, and rework the algorithm below to linearly interpolate from there.

Reply:
@Jaeyoung-Lim @jkflying I made the numbers as parameteres (risk_threshold_risk_based_speedup_, default_speed_) and modify speed_ calculation equation accordingly.